.circleci/config.yml

```
@@ -247,6 +247,30 @@ jobs:
       - run:
           name: Semantic Release
           command: npm run semantic-release
+  
+  run_diff_tests:
+    executor: node10
+    steps:
+      - setup
+
+      - run: mkdir junit
+
+      - run:
+        name: Load environment variables
+        command: cat env/TEST_DOMAIN >> $BASH_ENV
```

```suggestion
          name: Load environment variables
          command: cat env/TEST_DOMAIN >> $BASH_ENV
```

I think you need to indent this.